package.json

```diff
@@ -68,8 +70,7 @@
     "eslint-doc-generator": "^2.2.2",
     "eslint-plugin-markdown": "^5.1.0",
     "eslint-plugin-n": "^17.21.0",
-    "eslint-plugin-prettier": "^5.5.3",
-    "eslint-plugin-unicorn": "^56.0.1",
+    "eslint-plugin-unicorn": "^60.0.0",
```

Comment:
If there are a lot of fixes from the unicorn upgrade, it would be preferred to do it separately.

Reply:
Reverted the unicorn plugin update